stdlib/asm/math/u64.masm

```diff
@@ -129,6 +129,7 @@ end
 # The input values are assumed to be represented using 32 bit limbs, but this is not checked.
 # Stack transition looks as follows:
 # [b_hi, b_lo, a_hi, a_lo, ...] -> [c_hi, c_mid_hi, c_mid_lo, c_lo, ...], where c = (a * b) % 2^64
+# This takes 18 cycles.
```

Comment:
This was calculated by adding up the number of cycles each operation needs to run. Is correct?


| ops| cycles|
|----|----|
dup0..7         |  1
dup0..7         |  1
u32mul.unsafe   |  1
dup0..7         |  1
movup0..7       |  1
u32madd.unsafe	 | 1
swap	           | 1
movup0..7       |  1
dup0..7         |  1
u32madd.unsafe	 | 1
movup0..7       |  1
movup0..7       |  1
u32madd.unsafe	 | 1
movup0..7       |  1
movup0..7       |  1
u32add.unsafe	  |  1
movup0..7       |  1
add             |  1

| ops | sum_cycles
|---|---|
overflowing_mul | 18

Reply:
Yes, I believe this is correct.